tfhe/src/core_crypto/entities/polynomial.rs

```diff
@@ -7,7 +7,7 @@ use std::ops::{Index, IndexMut};
 /// A [`polynomial`](`Polynomial`).
 #[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct Polynomial<C: Container> {
-    data: C,
+    pub data: C,
```

Comment:
why ? this is not something we allow

Reply:
`FourierPolynomial` has a public `data` field
I'll try make both private

Reply:
Seems to complicated to make `FourierPolynomial.data` private
But I'll keep `Polynomial.data` private

Reply:
yep, that's what I was going to say for now fft module plays by different rules

Reply:
Done